distro/sql-script/src/test/java/org/camunda/bpm/sql/test/SqlScriptTest.java

```diff
@@ -0,0 +1,245 @@
+      "ACT_UNIQ_TENANT_MEMB_GROUP");
+
+  protected Properties properties;
+  protected Database database;
+  protected String databaseType;
+  protected String projectVersion;
+
+  @Before
+  public void setup() throws Exception {
+    InputStream is = getClass().getClassLoader().getResourceAsStream("properties-from-pom.properties");
+    properties = new Properties();
+    properties.load(is);
+
+    databaseType = properties.getProperty("database.type");
+    projectVersion = properties.getProperty("project.version");
+  }
+
+  @Test
+  public void shouldEqualLiquibaseChangelogAndCreateScripts() throws Exception {
+    // given
+    SnapshotParserFactory.getInstance().register(new DirectAccessSnapshotParser());
+    database = getDatabase();
+
+    executeSqlScript("create", "engine");
+    executeSqlScript("create", "identity");
+
+    try (Liquibase liquibase = getLiquibase()) {
+      //   snapshot created of the database for manual scripts
+      DatabaseSnapshot snapshotManualScripts = createCurrentDatabaseSnapshot();
+      //   database cleared and set up with Liquibase changelog
+      liquibase.dropAll();
+      liquibase.update(new Contexts());
+      //   snapshot created of the database for Liquibase changelog
+      DatabaseSnapshot snapshotLiquibaseChangelog = createCurrentDatabaseSnapshot();
+
+      //   diff created for both snapshot
+      DiffResult diffResult =  liquibase.diff(getDatabaseForSnapshot(snapshotManualScripts),
+          getDatabaseForSnapshot(snapshotLiquibaseChangelog), new CompareControl());
+
+      // when generating changes to apply between both databases
+      List<ChangeSet> changeSetsToApply = new DiffToChangeLog(diffResult, new CustomDiffOutputControl()).generateChangeSets();
+
+      // then
+      assertThat(changeSetsToApply).isEmpty();
+    } finally {
+      database = getDatabase();
+      try (Liquibase liquibase = getLiquibase()){
+        liquibase.dropAll();
+      }
+    }
+  }
+
+  protected Liquibase getLiquibase() throws URISyntaxException {
+    return new Liquibase("camunda-changelog.xml", getAccessorForChangelogDirectory(), database);
+  }
+
+  protected Database getDatabase() throws DatabaseException {
+    String databaseUrl = properties.getProperty("database.url");
+    String databaseUser = properties.getProperty("database.username");
+    String databasePassword = properties.getProperty("database.password");
+    String databaseClass = properties.getProperty("database.driver");
+    return DatabaseFactory.getInstance().openDatabase(databaseUrl, databaseUser, databasePassword, databaseClass,
+        null, null, null, new ClassLoaderResourceAccessor());
+  }
+
+  protected void executeSqlScript(String sqlFolder, String sqlScript) throws LiquibaseException {
+    String statements = IoUtil.inputStreamAsString(getClass().getClassLoader().getResourceAsStream(
+        String.format("sql/%s/%s_%s_%s.sql", sqlFolder, "h2", sqlScript, projectVersion)));
```

Comment:
Since we used `database.type` in other places as well, should we also make use of it here instead of `"h2"`?
```suggestion
        String.format("sql/%s/%s_%s_%s.sql", sqlFolder, databaseType, sqlScript, projectVersion)));
```

Reply:
That one is actually killing all other DBs right now 😄 🚀